lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -28,6 +28,8 @@
         &-message,
         &-no-message {
             font-size: var(--theme-body-1-font-size) !important;
+            white-space: normal !important;
```

Comment:
This changes the design please consult Shane Maciak if the change is okay from the design point of view. Also if the change is specific for ACA we should override this in ACA and not change the core component behaviour

Reply:
This change is not specific to ACA. On the kdesign change part, I can cross check with Shain once. Thanks.

Reply:
Checked with Shane, he seemed to agree with the change.